script/model/forecast.py

```diff
@@ -0,0 +1,585 @@
+#!/usr/bin/env python3
```

Comment:
Create a separate folder under `script` called `forecasting`, and put all the forecasting related code there. Create individual files for `DataLoader`, `QueryCluster`, `ForecastModel`, and `LSTM`. Rename this file to `forecaster.py`. Rename `ForecastTrainer` to `Forecaster`.